lib/unlocked/EditTile.dart

```diff
@@ -0,0 +1,485 @@
+                                              ),
+                                              title: Text('Browse Album'),
+                                            ),
+                                            ListTile(
+                                              leading: Icon(
+                                                Icons.search,
+                                                color: Color(0xff23036A),
+                                              ),
+                                              title: Text('Search Community'),
+                                            ),
+                                          ],
+                                        );
+                                      },
+                                    )
+                                  },
+                                  child: Image.asset(
+                                    widget.tile.imageUrl,
+                                    width: screenSize.width / 2,
+                                  ),
+                                ),
+                              ),
+                            ),
+                            Container(
+                              height: 24,
+                              child: Text(widget.tile.name,
+                                  style: TextStyle(
+                                    fontSize: 12.0,
+                                    fontFamily: "Robotto",
+                                    fontWeight: FontWeight.w500,
+                                  )),
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+            Card(
+              elevation: 0.0,
+              // margin: const EdgeInsets.fromLTRB(1.0, 8.0, 1.0, 8.0),
+              child: Column(
+                children: <Widget>[
+                  Container(
+                    decoration: BoxDecoration(
+                      border: Border(
+                        bottom: new BorderSide(color: Colors.black12),
+                      ),
+                    ),
+                    child: Column(
+                      children: [
+                        ListTile(
+                          // Tile Name
+                          title: Text('Tile Name',
+                              style:
+                                  TextStyle(fontSize: 14, color: Colors.black)),
+                          subtitle: Text(widget.tile.name,
+                              style: TextStyle(
+                                  fontSize: 12, color: Color(0xff7C7C7C))),
+                          trailing: Wrap(
+                            children: [
+                              Icon(Icons.keyboard_arrow_right,
+                                  size: 35.0, color: Colors.black54),
+                            ],
+                          ),
```

Comment:
Take a look at my `Typography` comment and see if you can apply some of that here!